internal/migrations/20210713123129_populate_infra_env.go

```diff
@@ -16,8 +17,9 @@ func populateInfraEnv() *gormigrate.Migration {
 			if err := tx.Exec("ALTER TABLE hosts ADD COLUMN IF NOT EXISTS infra_env_id text NULL;").Error; err != nil {
 				return err
 			}
+
 			// Populate the infra_env_id column
-			if err := tx.Exec("UPDATE hosts SET infra_env_id=cluster_id;").Error; err != nil {
+			if err := tx.Exec("UPDATE hosts SET infra_env_id=cluster_id where infra_env_id is null or infra_env_id = '';").Error; err != nil {
```

Comment:
Why does this need to change?

Reply:
It is incorrect to set the infra_env_id during migration if it already has a value

Reply:
How is this related to the go-swagger update though?

Generally I don't want to edit migrations unless we have a very compelling reason because they've already run in some environments and will not run again.

For example, I'd really expect to see a separate bug to make a change like this rather than slipping it into a PR about updating go-swagger.

Reply:
Actually I tackled into the problem while working on this.  You are right it is not a go-swagger issue.  Will remove